eng/scripts/Set-SkipJobVariables.ps1

```diff
@@ -0,0 +1,64 @@
+. "$PSScriptRoot/../common/scripts/common.ps1"
+$RepoRoot = $RepoRoot.Path.Replace('\', '/')
+
+# Set of files that don't require build or test when changed
+$skipFiles = @(
+    'CHANGELOG.md',
+    'README.md',
+    'SUPPORT.md',
+    'TROUBLESHOOTING.md',
+    'CONTRIBUTING.md',
+    'CODE_OF_CONDUCT.md',
+    'SECURITY.md',
+    'NOTICE.txt',
+    'LICENSE'
+)
+
+# Set of directories that don't require build or test when changed
+$skipDirectories = @(
+    '.github/',
+    'eng/images/',
+    'docs',
+    'Resources'
+)
+
+$skipBuildAndTest = $true
+
+$changedFilesScriptPath = "$RepoRoot/eng/common/scripts/get-changedfiles.ps1"
+$changedFiles = & $changedFilesScriptPath -DiffFilterType ''
+
+if ($changedFiles) {
+    $changedFiles | ForEach-Object {
+    Write-Host "Checking file: $_"
+    $fileName = [System.IO.Path]::GetFileName($_)
```

Comment:
Why do we need to get the full filename? Everything should already be relative to the repo root.